client/src/components/widgets/LoadingAlert.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <span class="alert alert-info">
+    Loading more information
```

Comment:
Maybe we could use the `fa-spinner` icon here? https://fontawesome.com/v4.7/icon/spinner

Reply:
https://user-images.githubusercontent.com/636801/125091810-c20cb780-e09e-11eb-9ba2-5b48450f1e1f.mov